src/main/java/com/ibm/watson/modelmesh/payload/RemotePayloadProcessor.java

```diff
@@ -47,16 +48,16 @@ public RemotePayloadProcessor(URI uri) {
 
     @Override
     public boolean process(Payload payload) {
-        return sendPayload(payload, prepareContentBody(payload));
+        return sendPayload(payload);
     }
 
     private static PayloadContent prepareContentBody(Payload payload) {
         String id = payload.getId();
         String modelId = payload.getModelId();
-        String kind = payload.getKind();
+        String kind = payload.getKind().toString().toLowerCase();
+        ByteBuf byteBuf = payload.getData();
         String data;
-        if (payload.getData() != null) {
-            ByteBuf byteBuf = payload.getData();
+        if (byteBuf != null) {
             final byte[] bytes = new byte[byteBuf.readableBytes()];
             byteBuf.getBytes(0, bytes);
             data = Base64.getEncoder().encodeToString(bytes);
```

Comment:
@tteofili wondering why you chose not to include my changes to use netty's own base64 support? The main advantage of that is that it's optimized direct buffer pooling will be used rather than allocating a large new byte array on the heap.

Reply:
for some reason that code block didn't work well when sending payloads, e.g., the base64 encoded String wasn't received correctly by https://github.com/trustyai-explainability/trustyai-explainability/blob/main/explainability-service/src/main/java/org/kie/trustyai/service/endpoints/consumer/ConsumerEndpoint.java

Reply:
@tteofili ok sure, thanks. Possibly I made a mistake, I will double check that code.

Reply:
@tteofili OK I think I know what the difference must be... maybe you could change it to this:
```java
            ByteBuf encoded = Base64.encode(byteBuf, byteBuf.readerIndex(), byteBuf.readableBytes(), false);
            //TODO custom jackson serialization for this field to avoid round-tripping to string
            data = encoded.toString(StandardCharsets.US_ASCII);
```
(as well as importing netty's `Base64` class instead of the `java.util` one)

Adding the `false` arg on the end will tell it to not insert newlines which should match the other encoder's behaviour.